### ISSUE TITLE
Default for MIDIAutoVoiceDumpOnPC should be 0 (OFF)

### ISSUE THREAD

**Issue by diyelectromusic** [probonopd/MiniDexed#611]
See discussion here: https://github.com/probonopd/MiniDexed/pull/581#issuecomment-1924560309

Basically any SysEx transmitted on voice change is also sent over any THRU link to a downstream MiniDexed/synth.  When selecting a performance, this will result in up to 8 seperate SysEx dumps all on MIDI channel 1 - as the default performances are configured all for MIDI channel 1.

This will make a mess of any downstream MiniDexed/DX7 as it will suddenly receive the sequence of SysEx dumps and thus change the "live" sounding voice to whatever the last active TG on the first MiniDexed selected as its voice in that performance...

Basically this setting makes no sense in a "performance"-selecting MiniDexed configuration.

Kevin
